Functions/TestResults.ps1

```diff
@@ -146,7 +157,7 @@ function Write-NUnitEnvironmentInformation($PesterState, [System.Xml.XmlWriter]
     $XmlWriter.WriteStartElement('environment')
 
     $environment = Get-RunTimeEnvironment
-    foreach ($keyValuePair in $environment.GetEnumerator()) {
+    foreach ($keyValuePair in $environment.GetEnumerator() | Where-Object -FilterScript {$_.Name -ne 'junit-version'}) {
```

Comment:
This might fail when user mocks `Where-Object`. It's not a big issue here, because at this point all Mocks should be removed, but pipeline is slow, and keeping track of where we can and cannot use cmdlets is difficult, so new code should rather not use cmdlets :) use `if  {$_.Name -eq 'junit-version'} { continue }` instead please. If there is no other option and you need to use cmdlet please use it via SafeCommands hashtable.